app/config/db.js

```diff
@@ -48,6 +49,8 @@ const dbConfig = {
   }
 }
 
+console.log(dbConfig.schema)
```

Comment:
Are you intending to output the schema value? If so, perhaps add the word 'schema' in your output

Reply:
Removed - Just added in to test it was using the right schema.